erpnext/patches/v4_0/reset_permissions_for_masters.py

```diff
@@ -16,5 +16,5 @@ def execute():
 		try:
 			reset_perms(doctype)
 		except:
-			print "Error resetting perms for", doctype
+			print("Error resetting perms for", doctype)
 			raise
```

Comment:
Oops. You are right. I was wrong. There's nothing wrong with that raise statement